doc/man1/openssl-mac.pod.in

```diff
@@ -66,20 +82,6 @@ Specifies the MAC key in hexadecimal form (two hex digits per byte).
 The key length must conform to any restrictions of the MAC algorithm.
 A key must be specified for every MAC algorithm.
 
-=item B<digest:>I<string>
```

Comment:
These options havent really disappeared from here :)

Reply:
Look further down :)  I moved them to the end.